### PR TITLE
Fix plane source

### PR DESCRIPTION
**Pull request by finetjul**

### CODE REVIEW

**Comment by finetjul** [comment]
@laurennlam

File: Sources/Filters/Sources/PlaneSource/index.js [hunk 1]
```diff
@@ -129,23 +129,22 @@ function vtkPlaneSource(publicAPI, model) {
...
+          vtkMath.cross(model.normal, n, rotationVector);
           theta = vtkMath.degreesFromRadians(Math.acos(dp));
         }
         // Create rotation matrix
         const transform = mat4.create();
         const negCenter = [];
-
         vec3.negate(negCenter, model.center);
 
-        mat4.translate(transform, transform, negCenter);
```
Comment (laurennlam) [inline Sources/Filters/Sources/PlaneSource/index.js:141]: `vec3.negate` doesn't modify model.center. Only `negCenter`. As `negCenter` is not used, you can remove `vec3.negate(negCenter, model.center)`

Comment (laurennlam) [reply Sources/Filters/Sources/PlaneSource/index.js:141]: My bad, didn't see it's used right after.

**Comment by laurennlam** [comment]
LGTM, thanks for the fix!

**Pull request merged**